src/ui/public/doc_table/components/table_row/details.html

```diff
@@ -1,6 +1,20 @@
 <td colspan="{{ columns.length + 2 }}">
-  <a class="pull-right" ng-href="#/doc/{{indexPattern.id}}/{{row._index}}/{{row._type}}/?id={{row._id | uriescape}}">
-    <small>Link to /{{row._index}}/{{row._type}}/{{row._id | uriescape}}</small></i>
-  </a>
+  <div class="documentTableRow__actions">
```

Comment:
Sorry for not catching this earlier, but we should use links instead of buttons here, since we're navigating away from this app, and users may want to open these links in new tabs. The icons don't work well with links, so we should probably take them out for now:

```html
  <div class="documentTableRow__actions">
    <a
      class="kuiLink documentTableRow__action"
      ng-href="{{ getContextAppHref() }}"
      ng-if="indexPattern.hasTimeField()"
    >
      View surrounding entries
    </a>
    <a
      class="kuiLink documentTableRow__action"
      ng-href="#/doc/{{indexPattern.id}}/{{row._index}}/{{row._type}}/?id={{row._id | uriescape}}"
    >
      View single entry
    </a>
  </div>
```

![image](https://cloud.githubusercontent.com/assets/1238659/22791950/edae54f2-ee9f-11e6-9acc-3b80db3de5f3.png)